README.md

```diff
@@ -451,6 +451,28 @@ Add the load module to the status modules list.
 set -g @catppuccin_status_modules_right "... load ..."
 ```
 
+### Git module
+
+#### Requirements
+This module depends on [gitmux](https://github.com/arl/gitmux).
+
+#### Install
+The preferred way to install gitmux is using [Binary release](https://github.com/arl/gitmux?tab=readme-ov-file#installing).
+[Download the latest](https://github.com/arl/gitmux/releases/latest) binary for your platform/architecture and uncompress it.
```

Comment:
Just link to the install section of gitmux since this may change

Reply:
Changed this to:
```
#### Install
To install gitmux, follow the instructions in the [gitmux documentation](https://github.com/arl/gitmux/blob/main/README.md#installing).
```

Let me know if this is correct.